<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SleepReminderClass</class>
 <widget class="QMainWindow" name="SleepReminderClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>220</width>
    <height>116</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>220</width>
    <height>116</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>220</width>
    <height>116</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>大毛早睡提醒</string>
  </property>
  <property name="windowIcon">
   <iconset resource="sleepreminder.qrc">
    <normaloff>:/SleepReminder/icon</normaloff>:/SleepReminder/icon</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <layout class="QGridLayout" name="gridLayout">
      <item row="2" column="0">
       <widget class="QPushButton" name="MaoDO">
        <property name="text">
         <string>开启 / 关闭</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="MaoNowTime">
        <property name="font">
         <font>
          <family>微软雅黑</family>
          <pointsize>22</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>设定时间：</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MaoHour">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>点</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="MaoMinute">
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>分</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="sleepreminder.qrc"/>
 </resources>
 <connections/>
</ui>
